java/client/src/main/java/glide/api/commands/SetBaseCommands.java

```diff
@@ -75,4 +75,19 @@ public interface SetBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> scard(String key);
+
+    /**
+     * Returns whether each member is a member of the set stored at <code>key</code>.
```

Comment:
I know this is from redis.io, but consider: 
```suggestion
     * Checks whether each member is contained in the members of the set stored at <code>key</code>.
```